src/components/Landing.js

```diff
@@ -27,17 +30,26 @@ class Landing extends React.Component {
     })
   }
 
+  handleHideArtworkChange(event) {
+    this.setState({
+      hideArtwork: event.target.checked
+    });
+  }
+
   componentDidMount() {
     window.ga('set', 'page');
     window.ga('send', 'pageview', window.location.pathname);
   }
 
   render () {
+    let dest = `/game/${this.state.username}/${this.state.period}`;
+    if(this.state.hideArtwork) dest += '/hard';
+
     return (
       <form className='landing'>
         <input onChange={this.handleUsernameChange} value={this.state.username} className='landing-username' type='text' placeholder='Last.FM Username' autoFocus />
 
-        <div className='landing-period'>
+        <div className='landing-section'>
```

Comment:
The `landing-section` naming is not clear enough IMO.. Maybe `game-options` or something like that?

Reply:
maybe `game-option`, singular?

Reply:
👍🏼